### ISSUE TITLE
Problem with Flutter 3.10

### ISSUE THREAD

**Issue by accvi** [Ahmadre/FlutterIconPicker#59]
Xcode build done.                                           45,7s
Failed to build iOS app
Error (Xcode): ../../../.pub-cache/hosted/pub.dev/flutter_iconpicker-3.2.2/lib/Dialogs/DefaultDialog.dart:129:23: **Error: 'SearchBar' is imported from both 'package:flutter/src/material/search_anchor.dart' and 'package:flutter_iconpicker/IconPicker/searchBar.dart'.**
DefaultDialog.dart:129
Could not build the application for the simulator.
Error launching application on iPhone 14 Pro.
Exited

**Comment by azimuthdeveloper** [comment]
Alright, so there were a few issues with this (otherwise good) package. I forked it and did the following:

* Removed the WhatsApp icon
* Fixed the Searchbar conflict

Because I have a verified dart pub page, I published flutter_icon_picker_plus with these issues addressed and bumped the version number. It's here: https://pub.dev/packages/flutter_iconpicker_plus.

I'm not trying to do a hostile takeover of this repo or anything, I just needed it fixed and I use it in my apps, and a few PR's were left unmerged. There's no guarentee in terms of long-term support for this forked version of icon picker (after all, maybe Ahmadre will return and merge some PR's). But in the meantime, feel free to use my fork of this.

If you want to kick in a few dollars to support the fork and publishing to pub.dev then feel free: https://www.buymeacoffee.com/flutterscratch
